src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js

```diff
@@ -54,6 +58,7 @@ export default class Option extends React.PureComponent<Props> {
         const classNames = classnames({
             [optionStyles.option]: true,
             [optionStyles.selected]: this.props.selected,
+            [optionStyles.hasCheckbox]: this.props.selectedVisualization === 'checkbox',
```

Comment:
Shouldn't that work more like the skins? So something like `[optionsStyle['visualization' + this.props.selectedVisualization]`? Then it would be easier to extend, in case we want radios or something similar one day...